app/controllers/index.rb

```diff
@@ -0,0 +1,32 @@
+require 'sinatra'
+
+get '/' do
+  'Eveanandi Blossom Butler'
+end
+
+get '/sealions' do
+  'This route lists all of the sealions'
+  Sealion.all.to_json
+end
+
+get '/sealions/:id' do
+  'This route finds a specific sealion'
+  Sealion.find(params[:id]).to_json
+end
+
+put '/sealions/:id' do
```

Comment:
This route is fundemenatlly correct, you might want to try deconstructing what update_attributes is doing here. ie. 

cur_lion = Sealion.find(params[:id]) 
cur_lion.name = "changed name"
cur_lion.save

It might help you understand a little better.